src/webgl/p5.Framebuffer.js

```diff
@@ -1479,6 +1479,37 @@ class Framebuffer {
    * @param  {Number} w width of the subsection to be returned.
    * @param  {Number} h height of the subsection to be returned.
    * @return {p5.Image} subsection as a <a href="#/p5.Image">p5.Image</a> object.
+   *
+   * @example
+   * <div>
+   * <code>
+   * // The `pixelColor` is logged to the console, returning red with full opacity.
+   *
+   * function setup() {
+   *   createCanvas(400, 400, WEBGL);
+   *
+   *   // Create an off-screen WebGL graphics buffer
+   *   let myBuffer = createGraphics(200, 200, WEBGL);
```

Comment:
Thanks for adding an example here! Right now this would be using the `p5.Graphics` `get` method instead of the `p5.Framebuffer` one. If we instead do this, I think this will be showing an example on the right class:
```js
let myBuffer = createFramebuffer({ width: 200, height: 200 })
myBuffer.draw(() => {
  background(0)
  noStroke()
  // etc
})

myBuffer.loadPixels();
// ...and then the rest from here looks good!
```
```

Reply:
Thank you ! The code has been updated.